_account-security/security/encryption/ssh-for-amazon-web-services.md

```diff
@@ -0,0 +1,231 @@
+
+  - item: |
+      {% include shared/ssh/ssh-intro-requirements.html type="requirements" requirement-type="windows-ssh-client" %}
+
+
+# -------------------------- #
+#        Instructions        #
+# -------------------------- #
+
+steps:
+  - title: "Verify the database's VPC"
+    anchor: "verify-database-vpc"
+    content: |
+      First, you'll log into AWS and verify the Virtual Private Cloud (VPC) the database is in. The SSH server you'll create in Step 2 must reside in the same VPC as the database. 
+
+      1. Log into your AWS account.
+      2. Navigate to the RDS Dashboard in AWS. If you use the **Services** menu (top left corner), click the **RDS** option under the **Database** section.
+      3. From the RDS Dashboard, click **Databases** on the left side of the page.
+      4. On the page that displays, click the database you're connecting to Stitch. This will open the Instance Details page.
+      5. Click the **Connectivity & security** tab if it's not already open.
+      6. Locate the **VPC** field in the **Networking** section:
+
+         ![The VPC field in the Instance Details page in AWS]({{ site.baseurl }}/images/shared/ssh/amazon-locate-vpc.png)
+
+      Keep the name of the VPC handy - you'll need it to complete the next step.
+
+  - title: "Create an SSH server in your VPC"
+    anchor: "create-ssh-server-in-vpc"
+    content: |
+      In this step, you’ll launch an EC2 instance to serve as the SSH server. This publicly accessible instance will act as an intermediary, forwarding the traffic from Stitch through an encrypted tunnel to the database in the private subnet.
+
+      **Note**: This instance must reside in the same VPC as the database. Refer to [Step 1](#verify-database-vpc) if you aren't sure which VPC to use.
+
+    substeps:
+      - title: "Configure the SSH server"
+        anchor: "configure-ssh-server"
+        content: |
+          The first part of creating an SSH server in your VPC is configuring the instance.
+
+          1. Navigate to the VPC Management Console in AWS. If you use the **Services** menu (top left corner), click the **VPC** option under the **Networking & Content Delivery** section.
+          2. On the VPC Dashboard, click the **Launch EC2 Instances** button.
+          3. Next, you’ll be asked to select the Amazon Machine Image, or AMI, that will be used to launch the instance.
+
+             We’ll be using a Linux-based AMI (like Ubuntu) for this tutorial:
+
+             ![Ubuntu Amazon Machine Image option in AWS]({{ site.baseurl }}/images/shared/ssh/amazon-ec2-ubuntu-ami.png)
+
+             Click the **Select** button next to the AMI you want to use.
+          4. On the next page, you'll select the instance type. Generally, a small instance will work just fine. For example: `t2.medium`. You can find more info about instance types on [Amazon's website](https://aws.amazon.com/ec2/instance-types/){:target="new"}.
```

Comment:
since "small" is also an instance size, recommend changing to "Generally, a small instance will work just fine. For example: `t2.small`.